index.html

```diff
@@ -143,15 +143,16 @@ <h2 class="contact-us" id="contact-me">CONTACT US</h2>
         <h2 class="contact-desktop">Let's chat!</h2>
         <p class="contact-quote">If you have an application you are interested in developing, a feature that you need
             built or a project that needs coding. I'd love to help with it </p>
-        <form action="https://formspree.io/f/xayvgowp" method="POST">
+        <form id="myForm" action="https://formspree.io/f/xayvgowp" method="POST">
             <ul class="my-form">
                 <li>
                     <label for="user-name">Full name</label>
                     <input type="text" name="user-name" id="user-name" placeholder="Full name" maxlength="30" required>
                 </li>
-                <li>
+                <li class="validatorContainer">
```

Comment:
- [ ] Great job adding your error message for "Emails written in Capital Letter", however pls as suggested by the previous reviewer, kindly add your error message close to your submit button and not above the textarea like you have done.

<img width="464" alt="Screenshot 2022-07-23 021929" src="https://user-images.githubusercontent.com/52934128/180584931-8957c1a4-d5bc-47cc-b576-de36abe6f8fe.png">